### ISSUE TITLE
Text Glitch

### ISSUE THREAD

**Issue by IdontCareJustHelpMe** [photopea/photopea#5757]
Hey,

I have this problem where I can't quite use the text tool. The problem is, is that when I type in the text, it becomes extremely small. I've tried changing pixels, size and shapes in all different ways, but can make it bigger or visible for that matter. All extensions are off, latest Chrome version, incognito mode just freezes when trying to upload a picture.

Thank you for your help

**Comment by camoody1** [comment]
My guess is that you're just doing it wrong.

**Comment by IdontCareJustHelpMe** [comment]
Yeah I think so too. I just managed to make it big by increasing pixels. But for some reason it's giving me 150px man on properties tab. Is there some way to increase this? When I somehow managed to upscale it, all was good. And when I tried sliding to make it smaller, it quickly jumped back to 150 and once again doesn't let me go up.

**Comment by camoody1** [comment]
If you want your text to be larger than size 150, don't use the slider. Just type in whatever size you want to use. The slider stops at 150, but you can type whatever size you want.
